### ISSUE TITLE
CryptoCat 3.2.03 constantly and regularly disconnecting from server

### ISSUE THREAD

**Issue by cats-meow** [cryptocat/cryptocat#177]
I just installed the CryptoCat 3.2.03 upgrade and will be downgrading to CryptoCat 3.2.02 since this newest version is constantly and regularly disconnecting from server.  The disconnection happens maybe every 15 seconds or so, have not timed it.

**Comment by cats-meow** [comment]
The reconnection happens automatically so the disconnection and reconnection is a continuous loop with regular sound pings.

**Comment by kaepora** [comment]
How am I supposed to help you with this? You've provided me with absolutely no debugging information and I clearly can't reproduce the issue.

**Comment by cats-meow** [comment]
I assumed that this issue would be reproducible by everyone.  I can try another test or two.

Interesting - I just disabled the WiFi network card in the computer and CC still thinks it is on-line !! ??
Then reconnected to same WiFi and after some time maybe 30 seconds, CC said disconnected.  Disconnection lasts maybe 5 seconds, then reconnects.  During the disconnection, if I watch a continuous ping in the terminal, the ping continues with no interruption, therefore it appears network connection is solid for ping.
Again I disabled the computer WiFi card, but CC never sees or knows that it is disconnected - CC thinks it is still on-line even though there is no Inet connection.

I now reconnect to Inet, but with a different hot spot, same computer WiFi card.  Ping is OK.  Do not see the problem.
So disable computer Wifi network card.  This is where I see a 2nd problem since CC thinks it is happy and connected to Inet, but this is impossible.

So now, I change to a different computer WiFi card and use the hot spot that seems to disconnect.  And I get disconnections again.  This means that my Samsung Note 3 as a hot spot seems to disconnect every minute for a few seconds.  So now I disable 2nd computer WiFi card and CC still thinks it is connected to Inet.

Summary:
Disconnection problem appears to be a defective Note 3 as hot spot.

Once CC is connected to Inet, if connection drops CC does not recognize this and thinks it is still on-line.

As further test, I re-enable computer WiFi card nr.2 and send I msg to partner.  Then I turn off the hot spot.  I hear no ping sound, and see no red bar saying I am disconnected.  I have no connection, but think everything is ok.   Ahhhh finally after some minutes, I see the red bar saying disconnected.

Conclusion:
1 disconnection is a local problem because defective hot spot
2 after losing Inet connection the red bar appears, but after a certain time.  I am testing and CC has no Inet connection since 5.5 minutes before red bar appears.  This is too long.

This ticket can be closed.
